app/src/main/AndroidManifest.xml

```diff
@@ -13,6 +13,10 @@
         android:supportsRtl="true"
         android:theme="@style/Theme.SummerPractice2022TvTracker"
         tools:targetApi="31">
+        <activity
```

Comment:
your code is not compilable , this LoginActivity does not exist yet, comment it or delete, but don't let uncompilable code. Also your branch is not up-to-date with main branch, there are new changes.
Please re-check the following text from Readme file:
![image](https://user-images.githubusercontent.com/88314300/178194552-fe8d83be-e1a3-4da4-93a0-efac359c9b9b.png)